tools/codegen/dest/register_dispatch_key.py

```diff
@@ -85,6 +98,65 @@ def __call__(self, f: Union[NativeFunctionsGroup, NativeFunction]) -> List[str]:
         else:
             assert_never(f)
 
+    def gets_generated_out_inplace_wrapper(self, f: NativeFunction, g: NativeFunctionsGroup) -> bool:
```

Comment:
This is the code to xref, I think

Reply:
yep thanks, this should be shared in `gen_external_aten_fallbacks.py`. Another thing to not worry about later when/if the boxed CPU fallback works out. (That's also why I kind of wanted to keep all the fallback stuff in a separate file, so it'll be easier to disentangle if we want to)